src/qt/walletcontroller.cpp

```diff
@@ -340,7 +349,11 @@ LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QW
 
 void LoadWalletsActivity::load()
 {
-    showProgressDialog(tr("Loading wallets…"));
+    showProgressDialog(
+      //: Title of progress window when which is displayed when wallets are being loaded.
+      tr("Load Wallet"),
+      //: Describing what is being done in background.
```

Comment:
```suggestion
      //: Descriptive text of the load wallets progress window which indicates to the user that wallets are currently being loaded.
```

Reply:
Addressed in ccecfa8

Reply:
Oh! Thanks for catching it! Addressed in 01bff8f